source/components/atoms/Input/Input.tsx

```diff
@@ -44,6 +44,12 @@ const StyledTextInput = styled.TextInput<InputProps>`
   color: ${({ theme }) => theme.colors.neutrals[0]};
   ${(props) => (props.center ? 'text-align: center;' : null)}
   ${(props) => (props.hidden ? 'display: none;' : null)}
+  ${({ editable, theme }) =>
+    editable === false &&
```

Comment:
Is there any reason for not using the logical not operator ie ```!editable``` ?

Reply:
No reason ^^ gonna change it and merge